test/index.js

```diff
@@ -560,4 +568,33 @@ describe('Marked renderer', () => {
       '<img src="http://bar.com/b.jpg" alt="a&quot;b" title="c&gt;d"></p>\n'
     ].join('\n'));
   });
+
+  describe('exec filter to extend', () => {
+    it('should execute filter registered to marked:renderer', () => {
+      const hexo = new Hexo(__dirname, {silent: true});
+      hexo.extend.filter.register('marked:renderer', function(renderer) {
```

Comment:
```suggestion
      hexo.extend.filter.register('marked:renderer', (renderer) => {
```

![image](https://user-images.githubusercontent.com/40715044/71348666-fb0e3580-25a7-11ea-932b-60d9f25f922a.png)

Please makes eslint happy.